### ISSUE TITLE
Accessibility audit: /patterns Sub navigation section heading level

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#306]
## Describe the issue
On /patterns the heading ‘Help users to...’ is a Level 3 but there's no level 2 heading to form a correct outline from.

## Fix recommendation
- [x] Either fix # and this is a non issue, or change the heading level to level 2.

## Related stories
- [Accessibility audit: Sub navigation section hidden text heading missing](https://github.com/dwp/design-system/issues/305)
